sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -408,14 +420,29 @@ private static OrderBy analyzeOrderBy(SelectAnalysis selectAnalysis,
         return new OrderBy(symbols, reverseFlags, nullsFirst);
     }
 
+    private static Symbol analyzeExpression(Expression expression,
+                                            ExpressionAnalyzer selectAnalysisAwareExpressionAnalyzer,
+                                            ExpressionAnalyzer expressionAnalyzer,
+                                            ExpressionAnalysisContext expressionAnalysisContext) {
+        Symbol symbol;
+        if(expression instanceof QualifiedNameReference) {
```

Comment:
The distinction is there because unless it's a QNR we want to analyze the symbol and not try to match it against the select analysis. (eg: `select details from ignored_nested order by details['column']` - we don't want to order by the details object but by the underlying column)